packages/components/src/patterns/supernumber/supernumber.stories.ts

```diff
@@ -0,0 +1,272 @@
+import '../../solid-components';
```

Comment:
I don't know if this is related to your [comment](https://github.com/solid-design-system/solid/pull/845#issuecomment-1983852520) or to chromatic but I had the behavior, that after switching too fast between the stories, none of them BUT the default story started counting on scroll. I had the reload the page to get it working again. 

If its only with chromatic, I am fine but otherwise this should be fixed... let me know if you need a screen sharing session to reproduce it.

Reply:
This is a chromatic limitation/quirk. When navigating through stories, the page is not reloaded so when new stories are run it assumed we are redeclaring const/let variables. This causes the js to crash. 
![image](https://github.com/solid-design-system/solid/assets/63849626/bf65485f-c186-4d4f-9796-cab5ab888a69)

This can be resolved by using the `var` keyword and scoping them globally. But that would not be ideal to put in a sample as `var` is not best practice. Especially since this issue is limited to Chromatic.

I am not sure whether we should use `var` but add a disclaimer or use const and still add a disclaimer.